### DISCUSSION TITLE
Layering an Image or Element under a Video Wallpaper

### DISCUSSION THREAD

**Discussion by Fatih120** [rocksdanister/lively#1761] [Q&A] [answered]
![image](https://github.com/rocksdanister/lively/assets/18276369/a433516f-9a1c-4fab-902e-7141cb30169c)

I have a looped video playing like this on my Desktop, but I would like to add a proper wallpaper _underneath_ the video so that the rest of the entire screen isn't black. I purposefully wanted the video to be smack-dab in the middle and small while the rest would be static, but so far it just seems to be the limitation of Lively. Assuming it's not already possible to somehow add a background via MPV configuration, would transparent layering be possible to add in the future?

**Comment by rocksdanister** [comment, marked as answer]
I don't know if mpv can do this, it is just a video player and not editor after all.

You can create a custom webpage video wallpaper with any number of layers:

https://github.com/rocksdanister/lively/wiki/Web-Guide-VI-:-Webpage-Video-Support
